webpack.config.js

```diff
@@ -137,7 +137,8 @@ module.exports = env => {
         '/etc': aemDomain,
         '/designations': aemDomain
       },
-      port: 9000
+      port: 9000,
+      public: "localhost.cru.org:9000"
```

Comment:
This is the value for localhost, but wouldn't be for other environments. Not sure if that matters.